.gitignore

```diff
@@ -31,3 +31,9 @@ cabal.sandbox.config
 
 # IDE project files
 .idea
+
+# Node (Javascript) modules
+/node_modules
+
+# TODO: eventually remove all public/js files from tracking
+/public/js/search
```

Comment:
It looks like he's only transpiling the search page for now based on the webpack.config file and their respective /public/js/* folder is where they would go. So I assume eventually as more pages get added, more will get added to the gitignore.